wezterm-gui/src/glyphcache.rs

```diff
@@ -4239,6 +4259,66 @@ impl<T: Texture2d> GlyphCache<T> {
                     );
                 }
             }
+            BlockKey::Braille(dots_pattern) => {
+                // `dots_pattern` is a byte whose bits corresponds to dots
+                // on a 2 by 4 dots-grid.
+                // The position of a dot for a bit position (1-indexed) is as follow:
+                // 1 4  |
+                // 2 5  |<- These 3 lines are filled first (for the first 64 symbols)
+                // 3 6  |
+                // 7 8  <- This last line is filled last (for the remaining 192 symbols)
+                //
+                // NOTE: for simplicity & performance reasons, a dot is a square not a circle.
```

Comment:
Did squares make a dramatic performance improvement in your demo?

Reply:
Ah I can see your comment in the PR itself (github didn't show me that you had commented there, only this change!). Sounds good!